launch.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash
+
+function assert_success() {
+    "${@}"
+    local status=${?}
+    if [ ${status} -ne 0 ]; then
+        send_notification "### Error ${status} at: ${BASH_LINENO[*]} ###"
+        exit ${status}
+    fi
+}
+
+xwiki_version="$XWIKI_VERSION" # for case if it will be set up from environment variable
+if [ -z "$xwiki_version" ]; then
+    xwiki_version="11.10.5"
+fi
+
+url="https://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-flavor-jetty-hsqldb/$xwiki_version/xwiki-platform-distribution-flavor-jetty-hsqldb-$xwiki_version.zip"
+filename="xwiki-platform-distribution-flavor-jetty-hsqldb-$xwiki_version"
+checkAddress=http://127.0.0.1:8080/xwiki/bin/get/Main
+
+assert_success mkdir xwiki
+assert_success cd xwiki
+
+assert_success curl "$url" -o xwiki.zip
+assert_success unzip xwiki.zip
+
+assert_success cd "$filename"
+assert_success chmod +x start_xwiki.sh
+
+assert_success ./start_xwiki.sh &
+assert_success curl --retry 5 --retry-delay 10 "$checkAddress"
```

Comment:
I don't think this retry is very useful, Jetty unfortunately does not accept input URL until it's fully started and this takes a little while. At least the first check will definitely fail. Would probably make more sense to put that after the sleep 60.

Reply:
Is it possible to make it infinity?

Reply:
https://curl.haxx.se/docs/manpage.html#--retry-max-time here said that you can replace `--retry` with `--retry-max-time`

Reply:
@InsanusMokrassar I think we should make the first call only after 60seconds.
Send Notification command is not found(line 7)

Reply:
> Is it possible to make it infinity?

The retry is fine like this IMO, it's just started way too early. I would just move the sleep before that line.

Reply:
> @InsanusMokrassar I think we should make the first call only after 60seconds.

Ok, so, you can add `sleep 60` on some line

Reply:
> @InsanusMokrassar I think we should make the first call only after 60seconds.

Ok, so, you can add `sleep 60` on some line, @somenath1435